CHANGELOG.md

```diff
@@ -1,6 +1,61 @@
 Change Log / Release Log for PIConGPU
 ================================================================
 
+0.3.1
+-----
+**Date:** 2017-10-11
+
+Field Energy Plugin, Gaussian Density Profile and Restarts
+
+This release fixes the energy field plugin diagnostics (not all field
```

Comment:
The energy plugin accessed wrong memory within the stack. The resulting error can not be estimated.

Maybe we should remove the description with `()`